README.md

```diff
@@ -187,4 +187,14 @@ Note: This image only provides the openimis frontend server. The full openIMIS d
     - run `yarn install`
 
 * after this you can execute `yarn start` and you should see local module in your imis application.
-  
\ No newline at end of file
+
+### To add github workflows files to the particular module
+* from `/openimis-fe_js`:
+  * run this command: `node dev_tools/addCiToModule.js <moduleNameDirectory>`
```

Comment:
Please change addCiToModule to addCIToModule.

Reply:
changed